.github/workflows/setup-helper.yml

```
@@ -0,0 +1,93 @@
+        
+        # CORS Configuration
+        CORS_ALLOWED_ORIGINS=http://localhost:3000,http://localhost:5173
+        CORS_ALLOWED_METHODS=GET,POST,PUT,DELETE,OPTIONS
+        CORS_ALLOWED_HEADERS=*
+        CORS_ALLOW_CREDENTIALS=true
+        EOF
+        
+        echo "✅ .env file generated successfully"
+    
+    - name: Upload .env as artifact
+      uses: actions/upload-artifact@v3
```

actionlint (1.7.7)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)